integration_tests/optout.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { expect, test } from '@playwright/test'
+import tokenVerification from './mockApis/tokenVerification'
+import { resetStubs } from './mockApis/wiremock'
+import feComponent from './mockApis/feComponent'
+import assessForEarlyRelease from './mockApis/assessForEarlyRelease'
+import { favicon, getSignInUrl, signIn, signOut, token } from './mockApis/auth'
+
+const baseUrl = 'http://localhost:3007'
+
+async function login({ authorities }: { authorities: string[] }, active = true) {
+  await Promise.all([
+    favicon(),
+    signIn(),
+    signOut(),
+    token({ authorities }),
+    tokenVerification.stubVerifyToken(active),
+    feComponent.stubFeComponentsFail(),
+  ])
+}
+
+test.describe('Opt out', () => {
+  test.afterEach(async () => {
+    await resetStubs()
+  })
+  test('Offender can opt out of HDC', async ({ page }) => {
+    const prisonNumber = 'A1234AE'
+
+    await feComponent.stubFeComponents()
```

Comment:
**suggestion(non-blocking):** I think we could move these into some shared setup as they shouldn't change for any tests other than the single test that checks "what if it doesn't work?"

Reply:
will do